### PR TITLE
troubleshoot #74 #75 

### PR DESCRIPTION
**Pull request by USFAkbari**
I am currently engaged in a detailed process of troubleshooting the Docker Compose file to identify and resolve various issues that may significantly impact its overall functionality and performance. This involves carefully examining the configuration to pinpoint any discrepancies or errors that could hinder its operation. In addition to this, I am also modifying the .env file to ensure that it aligns more closely with the specific requirements of the project. This alignment is crucial for the seamless operation of the application. To enhance the efficiency and effectiveness of these changes, I will be editing the Dockerfile as well, incorporating the necessary adjustments that are essential for optimizing the build process. It is particularly important to note that the mongosh command will need to be replaced with an alternative that better suits our current setup. Furthermore, I have taken the initiative to update several dependencies within the project to ensure optimal performance and functionality, which is vital for maintaining the integrity and reliability of the application as we move forward. This comprehensive approach will help us achieve a more robust and efficient deployment.

### CODE REVIEW

**Review by muety — CHANGES_REQUESTED** [review]
Please see my comments in #77.

**Comment by USFAkbari** [comment]
Ican understand!?? What comment

**Comment by USFAkbari** [comment]
I don't delete any dependency i just update them

**Comment by USFAkbari** [comment]
"By the way, please don't worry about this; I'm pleased to inform you that this milestone has successfully been achieved for my internship. It's a valuable learning experience, and I'm excited about the progress I've made so far."😀

**Comment by muety** [comment]
> I cant understand!?? What comment

Sorry, comment in #76.

File: .yarnrc.yml [hunk 1]
```diff
@@ -1,7 +1 @@
 nodeLinker: node-modules
-
-plugins:
```
Comment (muety) [inline .yarnrc.yml:3]: You removed the plugin here.

File: .env.example [hunk 2]
```diff
@@ -6,8 +6,8 @@ ANCHR_PUBLIC_URL=http://localhost:3000
 ANCHR_DB_USER=anchr
 ANCHR_DB_PASSWORD=
 ANCHR_DB_HOST=localhost
-ANCHR_DB_PORT=27017
-ANCHR_DB_NAME=anchr
```
Comment (muety) [inline .env.example:10]: Would prefer to stick to the default port and `anchr` as a user name.

File: README.md [hunk 3]
```diff
@@ -40,30 +40,33 @@ Anchr’s focus is on ease and quickness of use – short loading times, flat me
 ## 📡 How to run?
 ### Prerequisites
 In order to host Anchr on your own, you need a few things.
+
 * Node.js >= 20.x
-* MongoDB >= 6.x
-	* Alternative 1: [Mongo Atlas](https://mongodb.com/atlas) (hosted cloud MongoDB)
-	* Alternative 2: [FerretDB](https://www.ferretdb.io/) (with Postgres or SQLite)
+* MongoDB >= latest
```
Comment (muety) [inline README.md:45]: Would prefer to give a _minimum_ version here. Don't want to force people to always be at latest.

File: docker-compose.yml [hunk 4]
```diff
@@ -1,34 +1,51 @@
...
   anchr_db_data:
+
+networks:
+  devops_bridge:
+    driver: bridge
+    ipam:
+      driver: default
+      config:
+      - subnet:  172.18.0.0/29
+        gateway: 172.18.0.1
```
Comment (muety) [inline docker-compose.yml:51]: What's the purpose of this?

**Review by muety — CHANGES_REQUESTED** [review]

**Pull request closed without merging**